FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -161,7 +166,9 @@ - (FIRTrace *)activeTrace {
  * @return true if the platform could prewarm apps on the current device
  */
 + (BOOL)isPrewarmAvailable {
-  if (![[[GULAppEnvironmentUtil applePlatform] lowercaseString] isEqualToString:@"ios"]) {
+  NSString *platform = [[GULAppEnvironmentUtil applePlatform] lowercaseString];
+  if (![platform isEqualToString:@"ios"] && ![platform isEqualToString:@"maccatalyst"] &&
```

Comment:
I assume there should be macros to do this check much easily. Can you see if there are any platform MACROS?

Reply:
decided to go with the native `@available(iOS ..., *)` function.